discord/discord.c

```diff
@@ -93,14 +100,42 @@ static void handle_discord_spectate(const char* secret)
    RARCH_LOG("[Discord] spectate (%s)\n", secret);
 }
 
+static void handle_discord_join_response(void *ignore, const char *line)
+{
+   if (strstr(line, "yes"))
+      Discord_Respond(user_id, DISCORD_REPLY_YES);
+
+#ifdef HAVE_MENU
+   menu_input_dialog_end();
+   rarch_menu_running_finished();
+#endif
+}
+
 static void handle_discord_join_request(const DiscordUser* request)
 {
-   int response = -1;
-   char yn[4];
    RARCH_LOG("[Discord] join request from %s#%s - %s\n",
       request->username,
       request->discriminator,
       request->userId);
+
+   strlcpy(user_id, request->userId, sizeof(user_id));
+
+#ifdef HAVE_MENU
+      char buf[PATH_MAX_LENGTH];
```

Comment:
This needs to be declared at the top of the function for C89 backwards compatibility.

Reply:
done